### PR TITLE
Run bin/update using pattern-library_assets image

### PR DESCRIPTION
**Pull request by giorgiosironi**
Copying the files seems fine, but the second commit seems to have too many changes for such an update.

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
Running here produces _mostly_ the right result (using `pr-849` image, as there's been an update since this once - should really use the commit).

File: README.md [hunk 1]
```diff
@@ -70,6 +70,6 @@ var_dump($patternRenderer->getStyleSheets());
 Updating the library
 --------------------
 
-1. Install [Compass](http://compass-style.org/).
+1. Install [Docker](https://www.docker.com/).
 2. Execute `bin/update` to update the `resources` folder from Pattern Lab (you can pass a commit if needed, eg `bin/update baa62f0`).
```
Comment (thewilkybarkid) [inline README.md]: This will need to be the full sha, but we'll need to push them first. Need to mention PR tags too.

Comment (giorgiosironi) [reply README.md]: Is the sha stored somewhere? Or is it always passed as an input?

**Comment by giorgiosironi** [comment]
Let's merge and also try out the dependency update then.

**Comment by giorgiosironi** [comment]
https://alfred.elifesciences.org/job/dependencies-patterns-php-update-pattern-library/370/console updated nothing, which was expected; waiting to see future changes going through.

**Pull request merged**